palladian-commons/src/test/java/ws/palladian/helper/StopWatchTest.java

```diff
@@ -0,0 +1,192 @@
+package ws.palladian.helper;
+
+import static org.mockito.Matchers.anyLong;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import ws.palladian.helper.date.DateHelper;
+
+import java.lang.reflect.Method;
+
+@RunWith(PowerMockRunner.class)
+public class StopWatchTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+  @Rule public final Timeout globalTimeout = new Timeout(10000);
+
+  @PrepareForTest({System.class, StopWatch.class})
+  @Test
+  public void testGetStartTime() {
+
+    PowerMockito.mockStatic(System.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
```

Comment:
I'm a bit dubious about this test, part of me wanders if it even really needed. StopWatch.getStartTime() is just a call to System.currentTimeMillis() which should probably always be assumed to be correct. 

Technically there is nothing wrong with this test as we are proving that the return is always going to be equal to what our mocked system time class returns, but we are also effectively just proving that we can mock currentTimeMillis(); with no additional manipulation of the data happening in the logic of this method. 

This is probably a question of test usefuleness vs coverage gains. @JohnLBergqvist, @petersear @peterschrammel , @paulkennethkent What do you think?

Reply:
Well it covers part of their code...